onboarding_project/squad_pantry_app/models.py

```diff
@@ -71,12 +71,31 @@ def save(self, *args, **kwargs):
             self.closed_at = timezone.now()
         super(Order, self).save(*args, **kwargs)
 
+    def check_limit(self, limit, user_id):
+        """
+        check if the number of open order has reached the limit"
+
+        Keyword arguments:
+        self - object of the class order
+        limit - maximum number of possible open orders
+        user_id - id of logged in user
+        """
+        placed_orders = len(Order.objects.filter(status=self.ORDER_PLACED))
```

Comment:
Should we do a `len()` or a `.count()`? Which is more efficient?

Reply:
Count